user-guide/Getting_started/DaaS/Order_DataMiner_credits.md

```diff
@@ -27,20 +31,24 @@ To create and maintain a DaaS instance, your [organization](xref:Pricing_Usage_b
 
    1. Click *Review + subscribe*.
 
-      ![Azure portal SaaS resource](~/user-guide/images/Azure-SaaS-Resource.png)
+      ![Azure portal SaaS resource](~/user-guide/images/Azure_Portal_Basics.png)
+
+1. At the bottom of the window, click *Subscribe*.
+
+   ![Azure Portal SaaS resource subscribe](~/user-guide/images/Azure_Portal_Subscribe.png)
 
 1. Click *Configure account now*.
 
    This will redirect you to the DataMiner marketplace landing page.
 
    ![Configure account now](~/user-guide/images/Azure_Portal_Configure.png)
 
-1. Check whether the correct personal details are displayed, and adjust them if necessary.
-
-1. When all your personal details are correct, click *Purchase credits*.
+1. Click *Purchase credits*.
```

Comment:
The screenshot shows "Activate subscription", but the text says you need to click "Purchase credits". Is something missing here? Or should the text be updated?

Reply:
I updated to hastily, the text needs to be updated